pyproject.toml

```diff
@@ -4,7 +4,7 @@ build-backend = "setuptools.build_meta"
 
 [project]
 name = "caio"
-license = "Apache-2.0"
+license = {file = "COPYING"}
```

Comment:
Any particular reason to revert that here?
Setuptools will include license files which match these globs by default
```
['LICEN[CS]E*', 'COPYING*', 'NOTICE*', 'AUTHORS*']
```

Reply:
Just saw https://github.com/mosquito/caio/pull/42. That works too for license-files. However it omits the `License-Expression`.

Reply:
I don't see it as an either or but rather that both can be used together. Especially for simple license identifiers like `Apache-2.0`. Do note that for example PyPI will only display the `License-Expression` on the project page.
_The metadata will still include both or more specifically the license file will be included in the distribution._

Reply:
released as https://pypi.org/project/caio/0.9.24/

Reply:
> released as https://pypi.org/project/caio/0.9.24/

Thank you 🙏🏻